cli/object_command.go

```diff
@@ -301,6 +301,7 @@ func (c *objCommand) showBucketInfo(store nats.ObjectStore) error {
 			cols.AddRow("Maximum Bucket Size", humanize.IBytes(uint64(nfo.Config.MaxBytes)))
 		}
 	}
+	cols.AddRow("Storage Type", status.Storage())
```

Comment:
looks good, but lets make consistent with the KV one

https://github.com/nats-io/natscli/blob/3f6d70ab5c7c079a2ce3606b155fcc5a4e65925f/cli/kv_command.go#L787

Reply:
I'll update the string to just say "Storage", consistency is key